NFIQ2/NFIQ2Algorithm/src/features/FingerJetFXFeature.cpp

```diff
@@ -5,40 +5,37 @@
 
 #include <sstream>
 
-#define CBEFF (0x00330502)
-
 using namespace NFIQ;
 
-int cMem = 0;
-
-static void *
-m_malloc(size_t size, void *_)
-{
-	++cMem;
-	return malloc(size);
-}
-static void
-m_free(void *p, void *_)
-{
-	free(p);
-	--cMem;
-}
-
 FingerJetFXFeature::~FingerJetFXFeature()
 {
 }
 
 const std::string FingerJetFXFeature::speedFeatureIDGroup = "Minutiae";
 
+void
+FingerJetFXFeature::find_center_of_minutiae_mass(
+    std::unique_ptr<FRFXLL_Basic_19794_2_Minutia[]> &minutiaData,
+    unsigned int minutiaCount, int *x, int *y)
+{
+	int lx, ly, i;
+	lx = ly = 0;
+	for (i = 0; i < minutiaCount; i++) {
+		lx += minutiaData.get()[i].x;
+		ly += minutiaData.get()[i].y;
+	}
+	*x = lx / minutiaCount;
+	*y = ly / minutiaCount;
```

Comment:
I know this is pulled from BioMDI, but we could probably return a `std::pair` now. Just a thought.